pulser/simulation/simulation.py

```diff
@@ -92,12 +93,12 @@ def __init__(self, sequence, sampling_rate=1.0, evaluation_times=None):
             elif evaluation_times == 'Minimal':
                 self.eval_times = np.array([self._times[0], self._times[-1]])
             else:
-                raise ValueError("Wrong evaluation time label ")
+                raise ValueError("Wrong evaluation time label. It should "
+                                 "be `Full` or `Minimal`")
         elif isinstance(evaluation_times, (list, tuple, np.ndarray)):
-            #            all(isinstance(n, int) for n in lst)
             t_max = np.max(evaluation_times)
             t_min = np.min(evaluation_times)
-            if t_max > 1000 * self._times[-1]:
+            if t_max > self._times[-1]:
```

Comment:
If `t_max` is supposed to be in ns, why remove the `1000 *`? Isn't `self._times` in µs?

Reply:
You're right. I started off as ns and then thought it was better for it to be closer to `self._times`. I've now corrected it to be in μs